www/rules/community/index.html

```diff
@@ -0,0 +1,152 @@
+    <li>Set the country/region flag for each individual server in its server info, by setting the attribute <code>"flag"</code> to an integer representing the country code in ISO 3166-1 numeric format. On DDNet-based servers, the flag can be set with the <code>sv_flag</code> setting in the server configuration. A list of all available flags in ISO 3166-1 numeric can be found in the file <a href="https://github.com/ddnet/ddnet/blob/master/data/countryflags/index.txt"><code>data/countryflags/index.txt</code></a>.</li>
+    <li>Note that the community type filter may be revised or removed in a future version of the DDNet client. Community types are therefore automatically chosen based on the gametype of each individual server as backwards compatibility. Use a descriptive gametype name for each server so that filtering by gametype works.</li>
+  </ul>
+</div>
+
+<div class="block">
+  <h2 id="community-administration">Community Administration</h2>
+  <p>Open an Admin-Mail ticket on our <a href="/discord">Discord server</a> for any issues regarding community servers:</p>
+  <ul>
+    <li>to request creating a community for your servers (please read the above rules carefully and ensure that you satisfy them),</li>
+    <li>to update your own community’s information or icon (please also contact us in advance if you plan to shutdown your community),</li>
+    <li>to report community servers that break the above rules (please provide the server IP address and describe which rule was broken), and</li>
+    <li>for any further questions and concerns about the community rules and administration.</li>
+  </ul>
+</div>
+
+<div class="block">
+  <h2 id="list-of-communities">List of Communities</h2>
+  <div id="communities-table-container"><noscript><p>Please enable JavaScript to see the list of communities.</p></noscript></div>
+  <script type="text/javascript">
+    async function LoadCommunities() {
+      const Response = await fetch('https://info.ddnet.org/info');
+      if(!Response.ok) {
+        document.getElementById('communities-table-container').innerHTML = '<p>Failed to load community information.</p>';
+        return;
+      }
+      const Data = await Response.json();
+      let TableHtml = '<table class="communitiestable"><tr><th>Community</th><th>Contact (external links)</th></tr>';
+      for(const Community of Data['communities']) {
+        let Contact = Community['contact_urls'].map(Url => {
+          let HyperRef = !Url.startsWith('http') && Url.includes('@') ? 'mailto:' + Url : Url;
+          return `<li><a href="${HyperRef}">${Url}</a></li>`;
+        }).join('');
+        TableHtml += `<tr><td><img src="${Community['icon']['url']}"> ${Community['name']}</td><td><ul>${Contact}</ul></td></tr>`;
+      }
+      TableHtml += '</table>';
+      document.getElementById('communities-table-container').innerHTML = TableHtml;
+    }
+    LoadCommunities();
```

Comment:
Constructing HTML using JS is very brittle and can lead to injection attacks. In this case, it's unlikely because we control `https://info.ddnet.org/info`, but it's still bad practice.